doc/user/content/ingest-data/network-security/static-ips.md

```diff
@@ -31,7 +31,24 @@ connections from your region's egress IP addresses. You must allow connections
 from all egress IP addresses associated with your region. Connections may
 originate from any one of the egress IP addresses.
 
-To find the egress IP addresses associated with your region, you can query the
+## AWS Static Ips
```

Comment:
let's call them egress IPs rather than static

Reply:
also we label our regions like `aws/us-east-1` in the Console. maybe we just list out each region including the cloud provider in the table?